workflow/calculation/bb_sim.py

```diff
@@ -380,67 +390,96 @@ def unfinished():
-                ampdeamp(
-                    hf_acc[:, c],
-                    amp_function(
-                        bb_dt,
-                        n2,
-                        stat.vs,
-                        vs30,
-                        stat.vs,
-                        pga[c],
-                        fmin=fmin,
-                        fmidbot=fmidbot,
-                        version=site_amp_version,
-                    ),
-                    amp=True,
-                ),
-                bb_dt,
-                args.flo,
-                "highpass",
+        station_yaml = os.path.join(args.site_specific_dir, f"{stat.name}.yaml")
+        if os.path.isfile(station_yaml):
+            logger.debug(
+                f"Station {stat.name} has a site specific file. Running OpenSees"
             )
-            lf_acc[:, c] = bwfilter(
-                ampdeamp_lf(
-                    lf_acc[:, c],
-                    lf_amp_function(
-                        bb_dt,
-                        n2,
-                        lfvs30ref,
-                        vs30,
-                        stat.vs,
-                        pga[c],
-                        fmin=fmin,
-                        fmidbot=fmidbot,
-                        version=site_amp_version,
-                    ),
-                    amp=True,
-                ),
-                bb_dt,
-                args.flo,
-                "lowpass",
+            site_properties = SiteProp.from_file(station_yaml)
+            for j in range(3):
+                hf_filtered = bwfilter(
+                    hf_acc[:, j],
+                    bb_dt,
+                    args.flo,
+                    "highpass",
+                )
+                lf_filtered = bwfilter(
+                    lf_acc[:, j],
+                    bb_dt,
+                    args.flo,
+                    "lowpass",
+                )
+                hf_c = np.hstack((hf_start_padding_ts, hf_filtered, hf_end_padding_ts))
+                lf_c = np.hstack((lf_start_padding_ts, lf_filtered, lf_end_padding_ts))
+                bb_acc[:, j] = run_deconvolve_and_site_response(
+                    hf_c + lf_c, Components(j), site_properties, dt=bb_dt, logger=logger
+                )
+        else:
+            logger.debug(
+                f"Station {stat.name} does not have a site specific file. Running vs30 based amplification"
             )
-            hf_c = np.hstack((hf_start_padding_ts, hf_acc[:, c], hf_end_padding_ts))
-            lf_c = np.hstack((lf_start_padding_ts, lf_acc[:, c], lf_end_padding_ts))
-            if is_master and i == 0:
-                if len(hf_c) != len(lf_c):
-                    logger.critical(
-                        "padded hf and lf have different number of timesteps, aborting. "
-                    )
-                    comm.Abort()
-            bb_acc[:, c] = (hf_c + lf_c) / 981.0
+            pga = np.max(np.abs(hf_acc), axis=0) / 981.0
```

Comment:
Nothing wrong here, but I'd like a comment mentioning LF/HF acc is in cm/s/s, unlike BB (which is in g). My heart was racing when I saw this line...and I calmed down when I double checked qcore.timeseries code.

Reply:
Yeah, I've been having a few issues with that. LF/HF are cm/s/s, OpenSees needs m/s and BB is in g. Will make a comment on it though